packages/ffe-checkbox-react/src/Checkbox.js

```diff
@@ -1,23 +1,33 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { bool, node, string, func, oneOfType } from 'prop-types';
 import { v4 as hash } from 'uuid';
 import classNames from 'classnames';
 
 export default function CheckBox(props) {
-    const { children, inline, invalid, label, noMargins, ...rest } = props;
+    const {
+        children,
+        hiddenLabel,
+        inline,
+        invalid,
+        label,
+        noMargins,
+        ...rest
+    } = props;
 
     const id = props.id || `checkbox-${hash()}`;
     const labelProps = {
         className: classNames({
             'ffe-checkbox': true,
             'ffe-checkbox--inline': inline,
             'ffe-checkbox--no-margin': noMargins,
+            'ffe-checkbox--hidden-label': hiddenLabel,
```

Comment:
Is it too magic if this class and `hiddenLabel` value is derived from the fact that `aria-label !== undefined`?

Reply:
What if someone wants an image as a label?

Reply:
I suppose you could argue that `alt` on the image would work but I would personally still prefer this to not be magic.

Reply:
What about `--no-label`, to keep with the `--no-margin` modifier? Slightly different use case of course, but it's nice to have similar naming.

Reply:
I can change it to that if that's what you prefer, but I didn't want to encourage anyone to have a checkbox with no label what-so-ever so I went with hidden and added the bit about `aria-label`.

Reply:
The current name is fine by me - like Ivar says we don't want to encourage not using a label, even by name :)

Reply:
Yeah I don't feel strongly either way, Go with `hidden`